raven-app/src/components/layout/Sidebar/Sidebar.tsx

```diff
@@ -22,12 +22,14 @@ export const Sidebar = ({ children, isUserActive }: SidebarProps) => {
             <Stack>
                 <HStack justifyContent="space-between" spacing="3" h='33px'>
                     <Text fontSize="xl" fontWeight="semibold" ml='3'>Raven</Text>
-                    <IconButton
-                        size={"xs"}
-                        aria-label="Toggle theme"
-                        icon={colorMode === "light" ? <HiOutlineMoon /> : <HiOutlineSun />}
-                        onClick={toggleColorMode}
-                    />
+                    <HStack spacing={2}>
```

Comment:
why is there a HStack, with only one component inside? what is being stacked with what?